apps/src/common-styles.module.scss

```diff
@@ -0,0 +1,53 @@
+// As we move from inline css to css modules we are temporarily duplicating commonStyles.js into this css module.
+// Any changes in one should be made in the other to apply to all components.
+@import "color.scss";
+@import "style-constants";
```

Comment:
we can use the [`@use` keyword](https://sass-lang.com/documentation/at-rules/use) instead of `@import`, which is beneficial because it scopes the values rather than setting them as global imports